packages/ts/react-crud/src/autogrid.obj.css

```diff
@@ -53,7 +53,7 @@
   flex: 1 1 0;
   width: 120px;
 }
-.auto-grid-enum-filter vaadin-select-item {
+.auto-grid-enum-filter vaadin-select-value-button {
```

Comment:
As the select component now keeps the overlay as a child, this also targeted items in the overlay, but it should only target the selected value. Targeting the value button works as an alternative.